package.json

```diff
@@ -10,9 +10,9 @@
     "deploy-storybook": "build-storybook -o storybook_dist && gh-pages -d storybook_dist",
     "force-package-lock-https": "find . -type f -name \"package-lock.json\" | grep -v node_modules | xargs -I % sh -c 'echo \"Replacing http->https in %\"; sed -i \"\" -e \"s/http:\\/\\//https:\\/\\//g\" %'",
     "install:packages": "lerna exec -- npm install",
-    "postinstall": "npm run ci:packages",
+    "postinstall": "npm run ci:packages && npm run build",
```

Comment:
Isnt `npm run build` already a postinstall step?

Reply:
![image](https://user-images.githubusercontent.com/11341355/50291788-68ecc200-0467-11e9-805f-41928d27d8a8.png)
This causes the build to run twice